composer.json

```diff
@@ -31,6 +31,7 @@
     "require": {
         "php": "^7.4|^8.0",
         "symfony/cache": "^5.4|^6.0",
-        "symfony/contracts": "^2.5"
+        "symfony/contracts": "^2.5",
+        "phpmailer/phpmailer": "^6.6"
```

Comment:
Can https://symfony.com/doc/current/mailer.html be used instead?
That would make one less dependency in our code

Reply:
The basic SMTP methods in SymfonyMailer are private, so we cannot use them specifically: https://github.com/symfony/symfony/blob/6.1/src/Symfony/Component/Mailer/Transport/Smtp/SmtpTransport.php#L230

They are wrapped in the `send` command/